configs/accengage.json

```diff
@@ -1,30 +1,41 @@
 {
   "index_name": "accengage",
   "start_urls": [
-    "https://docs-alpha.accengage.com"
+    "https://documentation.accengage.com",
```

Comment:
The order of stat_urls matters. The first matching one is applied to the page, thus this object should be set as the last element as it matches every crawled page. I will do it and merge it